api/http/handler/endpoints/endpoint_agent_browse_docs.go

```diff
@@ -0,0 +1,25 @@
+package endpoints
+
+/// This feature is implemented in the agent API and not directly here.
+/// However, it's proxied.  So we document it here.
+
+// @summary Upload a file under a specific path on the file system of an environment (endpoint)
+// @description Use this environment(endpoint) to upload TLS files.
+// @description **Access policy**: administrator
+// @tags endpoints
+// @security ApiKeyAuth
```

Comment:
(maybe out of scope but will ask anyway). I imported this into postman and the auth initially has the key `Authorization` where the correct auth key for api tokens is `x-api-key` is there a way for the docs to start with the auth key of `x-api-key`? 
![image](https://user-images.githubusercontent.com/83188384/227353738-2c505271-daa5-4f8d-95b9-bb416395286f.png)